lib/Stats.js

```diff
@@ -1,24 +1,55 @@
 /*
-	MIT License http://www.opensource.org/licenses/mit-license.php
-	Author Tobias Koppers @sokra
+MIT License http://www.opensource.org/licenses/mit-license.php
+Author Tobias Koppers @sokra
 */
 "use strict";
 
 const RequestShortener = require("./RequestShortener");
 const SizeFormatHelpers = require("./SizeFormatHelpers");
 const formatLocation = require("./formatLocation");
 const identifierUtils = require("./util/identifier");
+const Tapable = require("tapable").Tapable;
+const SyncHook = require("tapable").SyncHook;
+const HookMap = require("tapable").HookMap;
+const SyncWaterfallHook = require("tapable").SyncWaterfallHook;
 
 const optionsOrFallback = (...args) => {
 	let optionValues = [];
 	optionValues.push(...args);
 	return optionValues.find(optionValue => typeof optionValue !== "undefined");
 };
 
-class Stats {
+const isTypeOf = (variable, type) => {
+	if (typeof variable === type) return true;
+	return false;
+};
+
+const assertTypeOf = (variable, type) => {
+	if (typeof variable === type || typeof variable === "undefined")
+		return variable;
+	throw new Error(
+		"Expected " +
```

Comment:
nit: Could you use a template string here please?

Reply:
You could run `yarn pretty-files` and let prettier handle it, even if it's _ugly_.
appveyor should be fine.

Reply:
I did exactly the same ( using vscode, automatic fix on save), but appevyor throws error, i guess there might be clash of two rules, 1 being the upper limit of characters per line, second being the whitespace before variable. Still will give that a shot.

Reply:
@ooflorent , should i change it? ran yarn pretty-files, same output, appveyor will shout for PR being bad :disappointed:

Reply:
Sorry, I meant Codacy when i said appveyor. It throws error, PR Quality is low. @ooflorent please review again.

Reply:
Well, keep it this way then.

Reply:
@ooflorent That's what I noticed too, prettier should be named `uglier` instead :D